src/rocdecode/vaapi/vaapi_videodecoder.cpp

```diff
@@ -85,17 +84,21 @@ rocDecStatus VaapiVideoDecoder::InitializeDecoder(std::string device_name, std::
             GetCurrentComputePartition(current_compute_partitions);
             if (current_compute_partitions.empty()) {
                 //if the current_compute_partitions is empty then the default SPX mode is assumed.
-                num_render_cards_per_socket = 8;
+                if (decoder_create_info_.device_id < visible_devices.size()) {
+                    offset = visible_devices[decoder_create_info_.device_id] * 7;
```

Comment:
Is this statement true in MI308 as well?

Reply:
@rrawther Yes, It is correct—the kernel driver always creates 8 render nodes per MI3xx family.